### PR TITLE
doc updated related to schedule schema

### PR DESCRIPTION
**Pull request by Sanjay1611**
This PR updates following - 

- all the docs related to  schedule as per change in its schema 
- docs related to schedule for halting and resuming schedule

Signed-off-by: Sanjay Nathani <sanjay.nathani@mayadata.io>

### CODE REVIEW

File: docs/chaosschedule-concepts.md [hunk 1]
```diff
@@ -29,30 +29,30 @@ This section describes the fields in the ChaosSchedule spec and the possible val
...
 </tr>
 <tr>
   <th>Default</th>
-  <td><code>now</code></td>
+  <td><code>n/a</code></td>
 </tr>
 <tr>
   <th>Notes</th>
-  <td>The <code>scheduleType</code> in the spec is a user defined flag to trigger type of schedule they want. Setting it to <code>now</code> ensures immediate formation of chaosengine. Setting it to <code>once</code> ensures formation of chaosengine at a specific time. Setting it to <code>repeat</code> ensures repeated formation of chaosengine within a specific time interval.</td>
+  <td>The <code>now</code> in the <code>spec.schedule</code> is a user defined flag to trigger type of schedule they want. Setting it to <code>true</code> ensures immediate formation of chaosengine.
```
Comment (ksatchit) [inline docs/chaosschedule-concepts.md]: > 
```suggestion
  <td>The <code>now</code> in the <code>spec.schedule</code> ensures immediate creation of chaosengine, i.e., injection of chaos.
```

File: docs/chaosschedule-concepts.md [hunk 2]
```diff
@@ -29,30 +29,30 @@ This section describes the fields in the ChaosSchedule spec and the possible val
...
 
 <table>
 <tr>
   <th>Field</th>
-  <td><code>.spec.schedule.executionTime</code></td>
+  <td><code>.spec.schedule.once.executionTime</code></td>
 </tr>
 <tr>
   <th>Description</th>
   <td>Time at which chaosengine is to be formed</td>
```
Comment (ksatchit) [inline docs/chaosschedule-concepts.md]:
```suggestion
  <td><code>.spec.schedule.once</code> refers to a single-instance execution of chaos at a particular timestamp specified by <code>.spec.schedule.once.executionTime</code></td>
```

File: docs/chaosschedule-concepts.md [hunk 3]
```diff
@@ -71,15 +71,15 @@ This section describes the fields in the ChaosSchedule spec and the possible val
 <table>
 <tr>
   <th>Field</th>
-  <td><code>.spec.schedule.startTime</code></td>
+  <td><code>.spec.schedule.repeat.startTime</code></td>
 </tr>
 <tr>
   <th>Description</th>
   <td>Flag to specify start of the range of time within which chaosengine is to be formed</td>
```
Comment (ksatchit) [inline docs/chaosschedule-concepts.md]:
```suggestion
  <td>Flag to specify start timestamp of the range within which chaos is injected, when the policy is <code>repeat</code>. The chaosengine is not created before this timestamp.</td>
```

File: docs/chaosschedule-concepts.md [hunk 4]
```diff
@@ -91,23 +91,22 @@ This section describes the fields in the ChaosSchedule spec and the possible val
 </tr>
 <tr>
   <th>Notes</th>
-  <td>When <code>startTime</code> is specified along with <code>.spec.schedule.type="repeat"
-  </code>. ChaosEngine will not be formed before this time, no matter when it was created.</td>
+  <td>When <code>startTime</code> is specified, ChaosEngine will not be formed before this time, no matter when it was created.</td>
```
Comment (ksatchit) [inline docs/chaosschedule-concepts.md]:
```suggestion
  <td>Flag to specify end timestamp of the range within which chaos is injected, when the policy is <code>repeat</code>. The chaosengine is not created after this timestamp.</td>
```

**Comment by ksatchit** [comment]
Minor comments to be addressed.

File: docs/chaosschedule-concepts.md (hunk 2 above)
Comment (Sanjay1611) [reply docs/chaosschedule-concepts.md]: Should I make this change in the `Notes` column instead of `Description` column?

**Review by ksatchit — APPROVED** [review]

**Pull request merged**